src/EmployerPR/SFA.DAS.EmployerPR.UnitTests/Application/Providers/Queries/GetProviders/GetRoatpProvidersQueryHandlerTests.cs

```diff
@@ -0,0 +1,25 @@
+﻿using AutoFixture.NUnit3;
+using FluentAssertions;
+using Moq;
+using NUnit.Framework;
+using SFA.DAS.EmployerPR.Application.Queries.GetRoatpProviders;
+using SFA.DAS.SharedOuterApi.InnerApi.Responses.RoatpV2;
+using SFA.DAS.SharedOuterApi.Interfaces;
+using SFA.DAS.Testing.AutoFixture;
+
+namespace SFA.DAS.EmployerPR.UnitTests.Application.Providers.Queries.GetProviders;
+public class GetRoatpProvidersQueryHandlerTests
+{
+    [Test, MoqAutoData]
+    public async Task Handle_ReturnCalendarEvents(
+        [Frozen] Mock<IRoatpV2TrainingProviderService> apiClient,
```

Comment:
variable name is wrong, should be service not api client

Reply:
yes, will fix